search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -748,6 +748,15 @@
     :revision_date
     :result-format})
 
+(def ^:private valid-deleted-granules-search-params
+  "Valid parameters for deleted granules search"
+  #{:revision-date
+    :provider
+    :parent-collection-id
+    :revision_date
```

Comment:
These should all be kebab-case (no underscores). Processing params should convert all underscores in query-params to dashes.

So removing :parent_collection_id and :revision_date should work without problem.

Reply:
This works the same as deleted collections does, I can swap the order of sanitize and validation to get it to work your way though.